pkgs/tools/misc/fetch-scm/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, guile }:
+
+let
+  file = "fetch.scm";
+  pkg = "fetch-scm";
+in
+  stdenv.mkDerivation rec {
+    pname = pkg;
+    version = "v0.1.5";
+
+    src = fetchFromGitHub {
+      owner = "KikyTokamuro";
+      repo = file;
+      rev = version;
+      sha256 = "sha256-H89VCNAYnTwVEqyInATvLHIB7ioe2zvIGTAM2MUo7+g=";
+    };
+
+    dontBuild = true;
+    dontPatchShebangs = true;
+
+    buildInputs = [ guile ];
+
+    installPhase = ''
+      runHook preInstall
+      # strip shebang, because nix removes `--no-auto-compile`
```

Comment:
the flag is not present in 0.1.5, I don't think nix removes it

Reply:
removed it, will make new release later